editoast/src/views/timetable/stdcm.rs

```diff
@@ -100,6 +99,8 @@ enum StdcmError {
     TrainSimulationFail,
     #[error("Path items are invalid")]
     InvalidPathItems { items: Vec<InvalidPathItem> },
+    #[error("Request validation failed: {message}")]
+    RequestValidationFail { message: String },
```

Comment:
How about a more direct (if you accept, please report everywhere: translation, etc.):
```suggestion
    #[error("Invalid request: {message}")]
    InvalidRequest { message: String },
```